### ISSUE TITLE
Can't login Board Administrator page not found[forum.joomla.org] 

### ISSUE THREAD

**Issue by chavab1** [joomla/joomla-websites#1225]
I know I shouldn't post this problem here but the Board Administrator page is gone and I couldn't find any other resource to help with login credentials.

Where do I go to recover my login credentials??

Thanx

**Comment by Sandra97** [comment]
Hi,

You can follow this tutorial https://docs.joomla.org/How_do_you_recover_or_reset_your_admin_password%3F or ask for help at https://forum.joomla.org

As this repo is only for issues with *.joomla.org properties, not about the Joomla CMS, please close this issue. Thanks!

**Comment by chavab1** [comment]
Yes, I knew I shouldn't have posted this here but I couldn't find any other resource. And I couldn't ask for help in the forum because I don't have my login credentials. Those are the login credentials that I'm missing, not my personal website's. I'm missing my credentials to the Joomla main site. I tried to do the recover but it doesn't recognize my username nor email. Yet, I tried to register again using the same info and it says they are already registered.

Thanx
